.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build artefacts
+on: [push, pull_request]
+concurrency:
+  group: test-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  sdist:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.7'
```

Comment:
No, vkgdr supports 3.7, and a wheel built against the oldest version will work with all newer versions.

Reply:
Oh, I see this is actually for the sdist. I think it makes no difference which version of Python is used to build the sdist, but I'd like to keep one consistent Python version through all the packaging files for vkgdr.

Reply:
Fair enough.

Reply:
Having said that, I ended up specifying 3.8 for readthedocs, but that's because doc/conf.py depends on importlib.metadata, which was only added in 3.8.